frontend/src/concepts/modelRegistry/apiHooks/useModelRegistryServices.ts

```diff
@@ -56,9 +60,16 @@ export type ModelRegistryServicesResult = {
   refreshRulesReview: () => void;
 };
 
-export const useModelRegistryServices = (namespace: string): ModelRegistryServicesResult => {
-  const [allowList, accessReviewLoaded] = useAccessReview({ ...accessReviewResource, namespace });
-  const [rulesReviewStatus, rulesReviewLoaded, refreshRulesReview] = useRulesReview(namespace);
+export const useModelRegistryServices = (
+  namespace: string | undefined,
+): ModelRegistryServicesResult => {
+  const [allowList, accessReviewLoaded] = useAccessReview(
+    { ...accessReviewResource, namespace: namespace || '' },
+    !!namespace,
+  );
+  const [rulesReviewStatus, rulesReviewLoaded, refreshRulesReview] = useRulesReview(
+    namespace || '',
+  );
```

Comment:
`namespace || ''` here is still kicking the can -- we still need to change `useRulesReview` so it works similarly to `useAccessReview`, the namespace passed to it can be undefined and it has an `if (shouldRunCheck) {` before it calls `checkAccess`. (although in that case it could just be `if (namespace) {` since that's the only thing that should stop it from its check unlike useAccessReview).